docs/plans/2026-04-09-phase-7-dx-trigger.md

```
@@ -0,0 +1,1166 @@
+        Err(ActionError::fatal("trigger does not accept external events"))
+    }
+```
+
+**Step 4: Run check + tests**
+
+Run: `cargo check -p nebula-action && cargo nextest run -p nebula-action`
+Expected: compiles, all tests pass
+
+**Step 5: Commit**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)